Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -154,12 +154,21 @@ protected override void MarshallRequest(InvokeFeatures features, APIGatewayProxy
                 requestFeatures.Method = apiGatewayRequest.HttpMethod;
 
                 string path = null;
+
+                // Replaces {proxy+} in path, if exists
```

Comment:
Seems like we should always be doing this logic regardless over the being a `{proxy+}`. It is possible to use this package and not use `{proxy+}` and use path parameters..

Reply:
Agreed with the above.

Reply:
I double checked and we are actually also using non-greedy parameters in non-proxy resource paths in our custom deployments that contain this PR.
This works just fine because the path does never get in the {proxy+} logic and gets replaced properly by line 175 
```
if (string.IsNullOrEmpty(path))
{
    path = apiGatewayRequest.Path;
}
```

I have added an additional test to cover this case, and it passes without any modifications to the PR